index.html

```diff
@@ -0,0 +1,192 @@
+      <ul>
+          <li><a href="#">CANOPY</a></li>
+          <li><i class="fa-sharp fa-solid fa-circle"></i></li>
+          <li><a href="#">Back End Dev</a></li>
+          <li><i class="fa-sharp fa-solid fa-circle"></i></li>
+          <li><a href="#">2022</a></li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tag">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JAVASCRIPT</li>
+        </ul>
+        <br>
+        <br>
+        <a href="https://atril-ben-weather-app.netlify.app/" class="btn">See Project</a>
+      </div>
+  </div>
+ </div>
+ </section>
+<section class="about_me">
+  <h3 class="about">About
+    Myself</h3>
+    <p class="about_datile">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <section class="contact_list1">
+      <h3 class="lets"><a href="#">LET’S CONNECT</a></h3>
+      <ul class="list_holder">
+          <li class="list"><a href="https://twitter.com/AtrilSyed"><i class="fa-brands fa-twitter"></i></a></li>
+          <li class="list"><a href="https://www.linkedin.com/in/syed-atril-831696248/"><i class="fa-brands fa-linkedin-in"></i></a></li>
+          <li class="list"><a href="#"><i class="fa-solid fa-m"></i></a></li>
+          <li class="list"><a href="https://github.com/Atril33"><i class="fa-brands fa-github"></i></a></li>
+          <li class="list"><a href="https://web.facebook.com/profile.php?id=100073724910623"><i class="fa-brands fa-facebook"></i></a></li>
+       </ul>
+      </section>
+      <button class="resume">Get my resume</button>
+ <section class="languages">
+  <h4>Languages</h4>
+  <div class="skills">
+    <div class="add_skill"><img src="Images/javascript.png"><h3>JAVASCRIPT</h3></div>
+    <div class="add_skill"><img src="Images/html.png"><h3>HTML</h3></div>
+    <div class="add_skill"><img src="Images/css.png"><h3>CSS</h3></div>
+  </div>
+  <div class="works">
+    <div class="work_skill"><h3>Frameworks</h3><img src="Images/next.png"></div>
+    <div class="work_skill"><h3>Skills</h3><img src="Images/next.png"></div>
+  </div>
+ </section>
+  </section>
+  <section class="full_form">
+    <div class="contact_me">
+      <h1>Contact me</h1>
+      <P>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</P>
+    </div>
+    <form action="/">
+      <input type="text" value="Syed Atril" placeholder="Your Name Here" id="input" required/>
+      <input type="email" value="youremail@gmail.com" placeholder="Your Email Here" id="input" required/>
+      <textarea id="textpanel" cols="30" rows="9" placeholder="Write Your Message Here"></textarea> 
+      <submit type="submit" class="submit">Get In Touch</submit>
```

Comment:
- [ ] I kindly suggest that your submit button is either an `input` element with type `submit` or a `button` element with type `submit` or with no `type`
- [ ] Kindly ensure that your form submits successfully after clicking the submit button because it currently can't submit with a `submit` element.